pom.xml

```diff
@@ -175,13 +175,14 @@
                         <exclude>CVE-2022-24823</exclude>
                         <exclude>CVE-2022-2047</exclude>
                         <exclude>CVE-2022-30187</exclude>
+                        <exclude>CVE-2022-38752</exclude>
```

Comment:
Add a comment with the dependency and a reason for this exclusion

Reply:
ok